R/quosure-class.R

```diff
@@ -0,0 +1,29 @@
+#' Reproducible class with environment and code.
+#'
+#' Reproducible class with environment and code.
+#' @name Quosure-class
+#' @rdname Quosure-class
+#' @slot code (`character`) to reproduce the environment
+#' @slot env (`environment`) environment which content was generated by the evaluation
+#'  of the `code` slot.
+#' @slot id (`integer`) random identifier of the code element to make sure uniqueness
+#'  when joining.
+#' @keywords internal
+setClass(
+  "Quosure",
+  slots = c(env = "environment", code = "character", id = "integer"),
```

Comment:
As highlighted before, I don't like `code` being a character field as there are more suitable data structures to keep the _code_. And yes, I am aware that this means no support for comments and I am OK with that.

Reply:
marking as resolved 
- PR here: https://github.com/insightsengineering/teal.code/pull/68